packages/query/src/index.ts

```diff
@@ -409,7 +412,7 @@ const generateQueryOptions = ({
       )?.slice(1, -1)}`
     : '';
 
-  if (!params.length) {
+  if (!params.length || isSuspenseQuery(type)) {
     if (options) {
       return `${queryConfig} ...queryOptions`;
     }
```

Comment:
This is because suspense queries don't expect to be passed the 'enabled' option (https://tanstack.com/query/latest/docs/framework/react/reference/useSuspenseQuery), which is passed a few lines down from here after this return statement. 
This failed a test, I do not know why this test hasn't failed before these changes but I think it somehow triggered this existing bug.

Reply:
Just a note for the person reviewing this, you can resolve this after reading if you want.